Components/Helper/Price.php

```diff
@@ -142,12 +142,17 @@ public static function calcArticlePriceInclTax(
             $priceRate = self::getProductPriceRateAfterDiscountForArticle($article, $shop);
             $value *= $priceRate;
         }
-        $tax = $article->getTax()->getTax();
-        $priceWithTax = $value * (1 + ($tax / 100));
+
+        //Check if Gross Price is displayed in frontend
+        if ($price->getCustomerGroup()->getTax()) {
```

Comment:
I'd add an extra check if `getCustomerGroup()` is available before calling `getTax()`